Spec/TestUtilities.swift

```diff
@@ -494,6 +495,52 @@ func getTestTokenDetails(key: String? = nil, clientId: String? = nil, capability
     return tokenDetails
 }
 
+func encodeString(_ input: String) -> String? {
+    let allowedQueryParamAndKey =  CharacterSet(charactersIn: "{}[]*: ")
+   if let encodedString_ = input.addingPercentEncoding(withAllowedCharacters: allowedQueryParamAndKey) {
```

Comment:
Indentation issues again

Reply:
Actually it was a leftover :) https://github.com/ably/ably-ios/pull/714/commits/54c84dea35bb680fbd7785571044c5385f71a4bd